test/functional/feature_dersig.py

```diff
@@ -110,7 +109,7 @@ def run_test(self):
             peer.sync_with_ping()
 
         self.log.info("Test that transactions with non-DER signatures cannot appear in a block")
-        block.nVersion = 3
+        block.nVersion = 4
```

Comment:
After changing this, the log message "Test that a version 3 block with..." should also be updated.

Reply:
Thanks, done.